lib/diagnostic.rb

```diff
@@ -141,5 +166,9 @@ def multiplier
     else
       1
     end
+
```

Comment:
To make the method chaining work, `punch`, `kick`, and `move` must each return `self.